recipe-server/normandy/base/templatetags/normandy_webpack_loader.py

```diff
@@ -0,0 +1,31 @@
+from django import template
+from django.utils.safestring import mark_safe
+
+from webpack_loader.templatetags.webpack_loader import get_files
+
+from normandy.base.utils import sri_hash
+
+
+register = template.Library()
+
+
+@register.simple_tag
+def render_bundle(bundle_name, extension=None, config='DEFAULT', attrs=''):
+    """
+    Modified version of webpack_loader's render_bundle that adds hashes
+    to tags for subresource integrity.
+    """
+    tags = []
+    chunks = get_files(bundle_name, extension=extension, config=config)
+    for chunk in chunks:
+        with open(chunk['path'], 'rb') as f:
+            chunk_hash = sri_hash(f.read())
+        if chunk['name'].endswith(('.js', '.js.gz')):
```

Comment:
I don't think we actually use `.js.gz` files, do we? That sounds like something we shouldn't do. I'd be fine if the extra bit wasn't here.